run.dev.bash

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+export BENTO_PUBLIC_STATIC_FILES_PATH=/bento-public/build/www
+
+cd /bento-public || exit
+
+ls -la
+
```

Comment:
Think we should add `npm run build-dev` here, fixed my issue where I only got 404 if I ran in local.
main.go uses a static file middleware to serve files in `build/www`, but this directory is ignored by git.
Adding `npm run build-dev` created the directory and files, after that no issues.